packages/website/docs/autocomplete-js.md

```diff
@@ -90,12 +90,12 @@ type ClassNames = {
   input?: string;
   completion?: string;
```

Comment:
I think this one still needs a better name. Maybe `hint` or `magic-placeholder`?

Reply:
I think we'll go with "prediction", but I'll remove this feature before going stable because I've got a way better and flexible API in mind. I'll take a bit more time to polish the feature.

Reply:
prediction is a better one!